config

```diff
@@ -10,7 +10,8 @@
 
 ngx_feature_name=
 ngx_feature_run=no
-ngx_feature_incs="#include <modsecurity/modsecurity.h>"
+ngx_feature_incs="#include <modsecurity/modsecurity.h>
+#include <stdio.h>"
```

Comment:
If I were you, I'd add a comment above the line explaining the issue. Otherwise, when someone revisits this in the future, they will not know why the line is there and whether it is safe to remove.

Reply:
I reviewed few Nginx module during I investigated this issue in Debian, but as I realized there isn't any comment over there. I think the PR's comment is enough here (which describes the issue and the solution too).

Btw, `printf()` needs `<stdio.h>`, that's the "normal" way. So far it has not been a problem because most C compilers added it implicitly.

Reply:
Ok, your call.